### PR TITLE
 First draft of heat exchanger tutorial

### PR DESCRIPTION
**Pull request by steveraysteveray**

### CODE REVIEW

**Review by MatthewSteen — COMMENTED** [review]
What is/will be the purpose of this page? Check https://diataxis.fr/ to confirm where it belongs. As is, it seems like it would be a better fit for Explanation or Reference.

**Comment by gtfierro** [comment]
I see the purpose of this page as explaining how heat exchangers are modeled in 223P, but it does not specifically walk you through the steps of doing that. I think this belongs under "explanation"

**Comment by steveraysteveray** [comment]
I'm fine with that - will make the adjustment to Explanation.

File: _toc.yml.orig
Comment (MatthewSteen) [inline _toc.yml.orig:1]: Looks like this file was added accidentally.

Comment (steveraysteveray) [reply _toc.yml.orig:1]: Oops, a vestige of the merge conflict resolution. Maybe I can just delete it directly online.

Comment (steveraysteveray) [reply _toc.yml.orig:1]: Yep, looks like it let me.

**Pull request merged**